src/DI.Abstractions/DI.Abstractions.csproj

```diff
@@ -7,7 +7,7 @@ Microsoft.Extensions.DependencyInjection.IServiceCollection</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>Microsoft.Extensions.DependencyInjection.Abstractions</AssemblyName>
     <RootNamespace>Microsoft.Extensions.DependencyInjection.Abstractions</RootNamespace>
-    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <NoWarn>$(NoWarn)</NoWarn>
```

Comment:
You can delete this line altogether. Also in DI.csproj too

Reply:
Oh I thought it's nice to keep them because it's easier to add warning exclusions later. But I guess that's not rocket science, so I'll remove.